plugins/forc-postgres/src/ops/forc_postgres_createdb.rs

```diff
@@ -159,7 +159,7 @@ pub async fn init(command: CreateDbCommand) -> anyhow::Result<()> {
 
     pb.finish();
 
-    info!("\n✅ Successfully created database at '{pg_db_uri}'.");
+    println!("\n✅ Successfully created database at '{pg_db_uri}'.");
```

Comment:
Same comment here regarding shortening existing commands if `- --verbose` is not set. Something like `✅ Database created` is fine if no verbosity is passed